cmd/simulacra/README.md

```diff
@@ -0,0 +1,70 @@
+# Simulacra 
+
+Simulacra is a command line tool that the oncall team can use to automatically spin up VMs with a customers environment. 
+
+Simulacra can read Ops Agent config files and infer all the third party apps that it needs to install. It then instantiates the VM according to user specifications, installs Ops Agent and then installs all the required third party applications. 
+
+We can specify various configurations for our desired VM. For example, we can specify the Operating System we want our VM to have. 
+
+Ultimately, we want to integrate Sigmulacra with our diagnostic tool and be able to generate Simulacra config files from the output of the diagnostic tool.  
+
+## Example Usage
+
+    > cat config.yaml
+        logging:
+            receivers:
+                redis:
+                    type: redis
+            service:
+                pipelines:
+                    redis:
+                        receivers:
+                            - redis
+
+    go run -tags=integration_test simulacra.go --ops_agent_config "/usr/local/google/home/shafinsiddique/ops-agent/integration_test/simulacra/config.yaml"
+
+
+This command invokes Simulacra to spin up a VM, install Ops Agent on it and then install the required third party applications. Based on the ops agent config file, the only third party application Ops Agent is configured with is Redis so Simulacra will only install Redis. 
+
+
+## Command Line Arguments
+
+For any advanced usage, Simulacra should be configured using a YAML file and then invoked by specifying the --config argument. 
```

Comment:
I would delete this line. Specifying that this is for quick one-offs in the next line is sufficient.